### PR TITLE
Lib support

### PR DESCRIPTION
**Pull request by VictorVictini**

### CODE REVIEW

File: struct.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+	tv_api.qssq = "qs_snapshoter_basic-symbol-quotes_" + tv_api.qsToken
+	tv_api.realtimeSymbols = make(map[string]bool)
+
+	// required responses for a given request being sent
+	// halts write requests from being sent until it is received
+	tv_api.halts = await_response{
+		requiredResponses: map[string]string{
+			"create_series":     "series_completed",
+			"modify_series":     "series_completed",
+			"request_more_data": "series_completed",
```
Comment (mohfez) [inline struct.go:89]: `"resolve_symbol":    "symbol_resolved",`

fixes history not loading sometimes

Comment (VictorVictini) [reply struct.go:89]: nice spot, I'll add it quickly

**Comment by mohfez** [comment]
lgtm

**Comment by Deadly-Pigg** [comment]
> Good work VictorVictini, I'll do all the work now.

so generous of you mohfez

**Comment by mohfez** [comment]
> Good work VictorVictini, Deadly-Pigg will do all the work now.

thanks for volunteering @Deadly-Pigg

**Comment by Deadly-Pigg** [comment]
> Good work VictorVictini, Deadly-Pigg will watch me do all the work now.

no problem for volunteering to watch you do all the work

**Comment by mohfez** [comment]
Looks like everything works fine now? i'll close the PR.

**Pull request merged**